scripts/linters/pylint_extensions.py

```diff
@@ -619,6 +619,7 @@ def visit_classdef(self, node):
             self.add_message('newline-below-class-docstring', node=node)
         elif second_line_after_doc == b'':
             self.add_message('newline-below-class-docstring', node=node)
+        self.check_docstring_structure(node)
```

Comment:
Few things:
1) Can we get away with the above 2 checks (if the same are covered in `check_docstring_structure`)?
2) There are other places where we do call this function again for classes having a constructor method - leading to repeated check of the same thing. Please think about how we can avoid that.

Reply:
I dont think we can get away with them as their not being covered in the `check_docstring_strucutre` also I dont think adding them to `check_docstring_strucutre` will be a good option either as they are class specific and `check_docstring_strucutre` is used for the fucntiondef node as well.

Are you taking `check_docstring_style` ?? if yes then its deleted in this PR as it is of no use now as its work is directly being done by the passing `classdef node` to `check_docstring_strucutre`.

Reply:
> I dont think we can get away with them as their not being covered in the `check_docstring_strucutre` also I dont think adding them to `check_docstring_strucutre` will be a good option either as they are class specific and `check_docstring_strucutre` is used for the fucntiondef node as well.

Okay, thanks! I just wanted you to check once. Actually there is PR currently trying to disallow new line after function docstrings so yeah we don't need to do it for functions.

> Are you taking `check_docstring_style` ?? if yes then its deleted in this PR as it is of no use now as its work is directly being done by the passing `classdef node` to `check_docstring_strucutre`.

Where is that deleted part which was doing checks for class docstrings? I don't see the change in this PR yet.

Reply:
let me check I thought I have deleted it.